spec/configureClientTesting.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
```

Comment:
i split up configuration for client testing from server testing for now to make the test suite work with the bundle